src/rust-crypto/rust-crypto.ts

```diff
@@ -281,64 +280,6 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, CryptoEventH
         return await this.eventDecryptor.attemptEventDecryption(event, this.deviceIsolationMode);
     }
 
-    /**
-     * Implementation of (deprecated) {@link MatrixClient#getEventEncryptionInfo}.
-     *
-     * @param event - event to inspect
-     */
-    public getEventEncryptionInfo(event: MatrixEvent): IEncryptedEventInfo {
```

Comment:
We're removing something that wasn't deprecated?

Reply:
https://github.com/matrix-org/matrix-js-sdk/blob/7d687533e753f02166a3014dcbddce07a8d1a4eb/src/common-crypto/CryptoBackend.ts#L77-L83

Already deprecated